src/components/ha-dialog.ts

```diff
@@ -141,10 +141,10 @@ export class HaDialog extends DialogBase {
 
       @media all and (max-width: 450px), all and (max-height: 500px) {
         .mdc-dialog .mdc-dialog__surface {
-          min-height: 100vh;
-          min-height: 100svh;
-          max-height: 100vh;
-          max-height: 100svh;
+          min-height: var(--mdc-dialog-min-height, 100vh);
+          min-height: var(--mdc-dialog-min-height, 100svh);
+          max-height: var(--mdc-dialog-max-height, 100vh);
+          max-height: var(--mdc-dialog-max-height, 100svh);
```

Comment:
I won't prefixc it with `mdc` it should be `--ha` prefixed

Reply:
That would be fine, but then the other current use of --mdc vars should change now as well.